### PR TITLE
Add noisetor bridge-01 to bridge_reachability list

### PR DESCRIPTION
**Pull request by patrickod**

### CODE REVIEW

**Comment by hellais** [comment]
@patrickod is this bridge shipped as part of Tor Browser?

**Comment by patrickod** [comment]
@hellais not yet but I've opened a trac issue for this @ https://trac.torproject.org/projects/tor/ticket/24756 so I guess we wait until it's shipped? Opened the PR at the same time to prevent it being forgotten.

File: bridge_reachability/tor-bridges-ip-port.csv [hunk 1]
```diff
@@ -151,3 +151,4 @@ host,port,nickname,protocol
 199.58.81.140,80,longclaw,dir_port
 204.13.164.118,443,bastet,or_port
 204.13.164.118,80,bastet,dir_port
+216.252.162.21,46089,noisebridge-01,obfs4
```
Comment (Em4E) [inline bridge_reachability/tor-bridges-ip-port.csv]: Tiny nit: the nickname here is `noisebridge-01`, but the bridge's actual nickname is `noisebridge01` (no dash).
```
Jan 08 19:10:11.000 [notice] new bridge descriptor 'noisebridge01' (cached): $0DB8799466902192B6C7576D58D4F7F714EC87C1~noisebridge01 at 216.252.162.21
```
Might also make sense to put the new bridge just after zipfelmuetze, so it's grouped with the other obfs4 bridges and not with the directory authorities.

**Comment by Em4E** [comment]
I hope OONI will be able to begin measuring this bridge soon. This is an important case to have because it's the first bridge that's added under Tor Browser's new build system, which may trip up censors whose enumeration depended on the old one.

**Comment by hellais** [comment]
@Em4E thanks for the reminder. I re-ordered the bridge line and am merging this now so we can begin testing it.

**Pull request merged**